crate_universe/private/vendor_utils.bzl

```diff
@@ -66,6 +73,7 @@ def crates_vendor_deps_targets():
         actual = select({
             ":linux_amd64": "@cargo_bazel.buildifier-linux-amd64//file",
             ":linux_arm64": "@cargo_bazel.buildifier-linux-arm64//file",
+            ":linux_ppc64le": "@cargo_bazel.buildifier-linux-amd64//file",
```

Comment:
This doesn't seem correct. The amd64 binary can run on a powerpc CPU?

Reply:
Hi @UebelAndre Apologies for the late response, I got assigned to another project.
This one does look like a mistake. But it suggests buildifier was not required at all when I built and tested on my local ppc64le VM. Wondering where it is being used with your Intel builds?